src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -44,6 +44,10 @@ public static void Register(HttpConfiguration config, WebHostSettings settings =
             var builder = new ContainerBuilder();
             builder.RegisterApiControllers(typeof(FunctionsController).Assembly);
             AutofacBootstrap.Initialize(builder, settings);
+
+            // Invoke registration callback
+            dependencyCallback?.Invoke(builder, settings);
```

Comment:
This gives tests the ability to easily hook into the DI registration process and inject dependencies while testing.